lib/datadog/tracing/distributed/headers/ext.rb

```diff
@@ -1,31 +1,33 @@
+# frozen_string_literal: true
 # typed: true
 
 module Datadog
   module Tracing
     module Distributed
       module Headers
-        # HTTP headers one should set for distributed tracing.
-        # These are cross-language (eg: Python, Go and other implementations should honor these)
+        # DEV-2.0: This module only exists for backwards compatibility with the public API. It should be removed.
+        # @deprecated use [Datadog::Tracing::Distributed::Ext]
         # @public_api
```

Comment:
We can't, as it will completely hide it from our [public API docs](https://s3.amazonaws.com/gems.datadoghq.com/trace/docs/Datadog/Tracing/Distributed/Headers/Ext.html) and that's technically a breaking change.

Best I can do is a stern comment for now :)

Reply:
@marcotc @ivoanjo

I'm pretty sure y'all implemented a breaking change anyway.

I've got a bot PR that broke trying to `Bump [ddtrace](https://github.com/DataDog/dd-trace-rb) from 1.6.1 to 1.7.0.` because `NameError: uninitialized constant Datadog::Tracing::Distributed::Headers`

Also, I could be wrong, but I don't think `Datadog::Tracing::Distributed::Ext` exists.

If it does, I'd appreciate a reference to it. ^^

Reply:
Apologies for the delay in answering @Judahmeek.

Most modules under `Datadog::Tracing::Distributed::Headers` are currently not considered part of the public API, as they are not tagged with `@public_api`.

Nevertheless, we have broken your setup, which is definitely not what we intended! 

Let us know if we can help you move to 1.7.0+, and also, if you can, can you share how you're using these modules so we can hopefully take that into consideration for future changes? That will help us evaluate the current API and if there's something we're currently not exposing and that we should.